lib/dataTrigger/dataTriggers/domainDataTrigger.js

```diff
@@ -0,0 +1,53 @@
+const DataTriggerExecutionResult = require('../DataTriggerExecutionResult');
+const DataTriggerExecutionError = require('../../errors/DataTriggerExecutionError');
+const hash = require('../../util/hash');
+
+/**
+ * @typedef trigger
+ * @param {Document} document
+ * @param {DataTriggerExecutionContext} context
+ * @return {Promise<DataTriggerExecutionResult>}
+ */
+async function domainDataTrigger(document, context) {
+  const rawDocument = document.getData();
```

Comment:
It's a not `rawDocument` which contains system fields as well. It's just a document data.